environments/staging/inventory.ini

```diff
@@ -46,5 +46,9 @@ hqriak00-staging.internal-va.commcarehq.org
 hqriak01-staging.internal-va.commcarehq.org
 
 [control]
+control.internal-va.commcarehq.org
 
 [pg_standby]
+
+[ansible_skip:children]
```

Comment:
Do you see this being used elsewhere? It feels like this adds a bit of complexity for something that's fairly unnecessary.

Any reason to not solve this by making a different staging server control?

Reply:
Yeah, I don't know, but I could totally see a third party setting up a number of different monoliths for different customers and using the control machine for all of them. Not that that's great security-wise. But yeah, I don't know. Just for the specific problem at hand, this seemed like an easier solution to me than setting up a new server and telling everyone to set up either control machine envs again.

Reply:
> setting up a new server

I was originally thinking just reusing an already used server, not provisioning a new one. (we reuse kafka0 as the "control" on ICDS, so there's precedent)

Anyways, I'm likely not the person to be commenting on this since I'm still stuck in my ways and only use cchq for ansible deploying stack